### PR TITLE
change to disable search message

### PR DESCRIPTION
**Pull request by Anxhela21**
**Related Issue:**  open-cluster-management/backlog#12024

### Description of changes
- minor change to disabled search message.

### CODE REVIEW

**Review by jlpadilla — CHANGES_REQUESTED** [review]
How about `Search is disabled on some clusters.` and `View clusters with search add-on disabled.`

@dockerymick Can you review? The first message must be as short as possible because of UI placement.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

Approving, but would like to get @dockerymick approval before merging.

This change is very low risk and improves the user experience, so it's good to merge at this stage of the release.

**Comment by SherinV** [comment]
Thanks @Anxhela21 for making this change.

File: frontend/public/locales/en/search.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "search.saved.searches.section.title": "Saved searches",
     "search.suggested.searches.seaction.title": "Suggested search templates",
     "searchbar.button.save.search": "Save search",
-    "messages.S20.short": "More on disabled clusters",
+    "messages.S20.short": "Search is disabled on some clusters",
```
Comment (dockerymick) [inline frontend/public/locales/en/search.json:55]: Add a period to both statements

**Review by dockerymick — APPROVED** [review]
Left a comment. Otherwise this looks good to me!

**Pull request merged**